src/Language/PureScript/Docs/Prim.hs

```diff
@@ -127,6 +129,18 @@ primSymbolDocsModule = Module
   , modReExports = []
   }
 
+primNatDocsModule :: Module
+primNatDocsModule = Module
+  { modName = P.moduleNameFromString "Prim.Nat"
+  , modComments = Just "The Prim.Nat module is embedded in the PureScript compiler. Unlike `Prim`, it is not imported implicitly. It contains automatically solved type classes for working with `Nats`."
```

Comment:
Rather than "working with `Nats`", should this say, "working with type-level natural numbers"?

Reply:
:+1:, although I originally based this off the documentation for `Prim.Symbol`, so maybe that should be reworded in the future too, likely after #3103